packages/expo-checkbox/src/Checkbox.types.ts

```diff
@@ -15,6 +15,4 @@ export interface CheckboxProps extends ViewProps {
   onValueChange?: (value: boolean) => void;
 }
 
-export type CheckboxComponent = React.FC<CheckboxProps> & {
-  isAvailableAsync: () => Promise<boolean>;
```

Comment:
maybe we should add this method back temporarily and have it just return `true` and a warning about it being deprecated and to be removed soon, just in case people are using it -- better to get a warning than crash